moveit_setup_assistant/include/moveit/setup_assistant/tools/moveit_config_data.h

```diff
@@ -282,10 +280,10 @@ class MoveItConfigData
   // ******************************************************************************************
 
   /// Load a robot model
-  void setRobotModel(robot_model::RobotModelPtr robot_model);
+  void setRobotModel(moveit::core::RobotModelPtr robot_model);
```

Comment:
you're usually more structured with changes like this. I would have expected that in a separate commit.